lib/i18n/tasks/translators/yandex_translator.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'i18n/tasks/translators/base_translator'
+
+module I18n::Tasks::Translators
+  class YandexTranslator < BaseTranslator
+    def initialize(*)
+      begin
+        require 'yandex-translator'
+      rescue LoadError
+        raise ::I18n::Tasks::CommandError, "Add gem 'yandex-translator' to your Gemfile to use this command"
+      end
+      super
+    end
+
+    protected
+
+    def translate_values(list, **options)
+      list.map { |item| translator.translate(item, options) }
```

Comment:
Note: This is going to cause lots of requests to the Yandex API, processed in-sequence (slow).

It would be better to pass multiple `text` parameters to the request, batched to avoid hitting the 10,000 character limit, each batch in its own Thread or Fiber. This should be implemented in the yandex gem.